src/gui/pedalboard/editors/Port.cpp

```diff
@@ -1,10 +1,26 @@
 #include "Port.h"
 #include "../cables/CableDrawingHelpers.h"
 
-Port::Port (const Colour& processorColour) : procColour (processorColour)
+Port::Port (const Colour& processorColour, const PortType type) : procColour (processorColour), portType (type)
 {
 }
 
+Colour Port::getPortColour()
+{
+    if (portType == modulation)
+    {
+        return Colours::rebeccapurple;
```

Comment:
I'm not totally sold on this colour, I was thinking something blue/purplish as those seem to be the go to for modulation.